contracts/contracts/token/OUSD.sol

```diff
@@ -277,41 +278,41 @@ contract OUSD is Governable {
         returns (int256 rebasingCreditsDiff, int256 nonRebasingSupplyDiff)
     {
         RebaseOptions state = rebaseState[account];
-        int256 currentBalance = int256(balanceOf(account));
-        uint256 newBalance = (currentBalance + balanceChange).toUint256();
```

Comment:
Why did this change away from `uint256`? A balance is a number that can only be positive. It also looks like this changes makes us have to do many more conversions later in the code.

Reply:
because of the check on line 283. Though you are right this could be done in a cleaner way. Done here: https://github.com/OriginProtocol/origin-dollar/pull/2306/commits/21f7eb760ff4a50f78371a30b2ab48db133e4dc4

Reply:
Thanks! Good catch.